### ISSUE TITLE
sex chromosome turnover

### ISSUE THREAD

**Issue by ASEMBER2** [Tree-of-Sex/toso#66]
### In which category is your new term request?

- [X] new_definition
- [X] existing_definition

### Preferred term label

sex chromosome turnover

### Synonyms

transition between sex chromosome systems

### Textual definition

A transition from original to another system of sex chromosomes. This term may encompass the following situations: i) a change in the identity of the master sex-determining (MSD) gene, ii) a change in the linkage group/synteny block representing the sex chromosomes and/or iii) change in the type of heterogamety (i.e. transition from XY to ZW system or vice versa, which may occur even within the same linkage group). Another specific situation is that a sex chromosome turnover can proceed via sex chromosome–autosome fusion where the original MSD gene is usually preserved but novel (previously autosomal) segment becomes sex-linked. Above mentioned mechanisms may contribute to restart the sex chromosome degeneration/differentiation process (another mechanism is "fountain of youth"). More broadly, turnover may mean also the transition from ESD to GSD and vice versa (transition between sex determination modes).

### Suggested parent term

sex determination
sex chromosome GO:0000803

### Attribution

0000-0003-4441-9615

### Additional context

Related terms:

co-option of sex chromosomes (independent use of the same synteny blocks/linkage groups for the role of sex chromosomes in related species/lineages). The reported cases of repeated reuse of the same synteny blocks led to a definition of “limited options” theory. The explanation would be that some chromosomes are, due to their genetic composition or recombination frequency, predisposed to become sex chromosomes in the course of evolution. Either they carry one of the genes rereatedly used for creation of master sex-determining gene, or they are enriched with sexually-antagonistic genes, or they have a naturally low frequency of recombination.

Homologous transition = sex chromosome turnover which occurs within the same linkage group
Heterologous transition = different pair of chromosomes represents new sex chromosomes.
Cis-heterogamety = the type of heterogamy is maintained after the transition (ZW to ZW, XY to XY)
Trans-heterogamety = the type of heterogamy changes during the transition (XY to ZW and vice versa)

Suggested literature:

Saunders PA. 2019 Sex chromosome turnovers in evolution. eLS. (doi:10.1002/9780470015902.
a0028747)

Vicoso B (2019) Molecular and evolutionary dynamics of animal sex-chromosome turnover. Nat Ecol Evol. 3(12):1632–1641. https://doi.org/10.1038/s41559-019-1050-8

Meisel RP (2020) Evolution of sex determination and sex chromosomes: a novel alternative paradigm. BioEssays 42(9):1900212. https://doi.org/10.1002/bies.201900212

Kitano, J., Ansai, S., Takehana, Y., & Yamamoto, Y. (2024). Diversity and Convergence of Sex-Determination Mechanisms in Teleost Fish. Annual Review of Animal Biosciences, 12(1), 233-259. https://doi.org/10.1146/annurev-animal-021122-113935

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by KamilSJaron** [comment]
First, the definition is by far to lengthly - it could (potentially, not necessarily) list the mechanisms of turnovers, but not theories or explanations (like the fountain of youth)

We (Kamil, @amakunin Anya) discussed that this concept is challenging to define it properly - at what point we call a sex chromosome a new chromosome? yes, If it fuses with a different chromosome, but what if it has a large insertion, or expansion of repeats, is it considered still the same chromosome? Perhaps listing mechanisms would help addressing this, but we should have that list meaningful and comprehensive.

The three we came up with are:
- change of MSD locus to a different linkage group
- translocation of the SD to a different linkage group
- change of linkage groups which include the previous sex chromosome

Kamil's comment to the current categorisation, the mechanism 3 is a type of mechanism 1 (ZY -> ZW must include a change of MSD locus, right?).

**Comment by astridboehne** [comment]
I agree with the comment above; for me turnover would require the change of chromosomal location; unsure how to call a change in heterogamety, maybe just "heterogamety transition"?

**Comment by tilia2501** [comment]
In relation to Kamil's comment above, in the Vicoso 2019 review (https://doi.org/10.1038/s41559-019-1050-8), the sex chromosome turnover is defined as:

"Replacement of the ancestral sex chromosomes by a new set of sex-determining chromosomes."

I think this simple definition encompasses the three scenarios above?

**Comment by lukkrat** [comment]
evolutionary switch from one sex chromosome system to another by the emergence of a new sex-determining locus or a translocation of the sex-determining locus to another chromosome

definition following https://royalsocietypublishing.org/doi/10.1098/rstb.2020.0426
